docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -367,11 +311,44 @@ min-gas-price="1337"
 The minimum price that a transaction offers for it to be included in a mined block.
 The default is 1000.
 
-### network-id
+### network
 
-!!!important
-    This option is deprecated in favor of the new `--network` option.
-    It will be completely removed in the 0.9 release.
+```bash tab="Syntax"
+--network=<NETWORK>
+```
+
+```bash tab="Example Command Line"
+--network=rinkeby
+```
+
+```bash tab="Example Configuration File"
+network="rinkeby"
+```
+
+Known network predefined configuration.
+The default is `mainnet`.
+
+Possible values are :
+
+`mainnet`
+:   Main Ethereum network
+
+`ropsten`
+:   PoW test network similar to current main Ethereum network. 
+
+`rinkeby`
+:   PoA test network using Clique _(only compatible with some clients)_.
```

Comment:
I'm not sure if 'only compatible with some clients' is strictly accurate -> they can all run on Rinkeby with a rinkeby genesis file can't they?  Also, we don't specifically doc areas where Pantheon differs from the other clients (account management being the exception).   We're documenting Pantheon, not what's possible/not possible on the other clients.  Let me know if you want to discuss.